queue_classic_admin.gemspec

```diff
@@ -22,7 +22,6 @@ Gem::Specification.new do |s|
   s.add_dependency "pg"
   s.add_dependency "will_paginate", ">= 3.0.0"
   s.add_dependency "will_paginate-bootstrap", ">= 0.2.0"
-  s.add_dependency "twitter-bootstrap-rails"
```

Comment:
Because scheduling was added in 3.1 and I don't see any condition or anything to prevent stuff from crashing? Or did I miss it? :)